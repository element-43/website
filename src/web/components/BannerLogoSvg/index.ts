export { default as BannerLogoSvg } from './BannerLogoSvg';
