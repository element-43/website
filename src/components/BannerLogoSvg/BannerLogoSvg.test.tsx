import { shallow, ShallowWrapper } from 'enzyme';
import React from 'react';

// Components.
import { BannerLogoSvg, IProps } from './BannerLogoSvg';

interface IScope {
  props: IProps;
  wrapper: ShallowWrapper;
}

describe('<BannerLogoSvg />', () => {
  let scope: IScope;

  beforeEach(() => {
    const props: IProps = {};

    scope = {
      props,
      wrapper: shallow(<BannerLogoSvg {...props} />),
    };
  });

  describe('<BannerLogoSvg /> snapshots', () => {
    it('should match the snapshot with the default properties', () => {
      expect(scope.wrapper).toMatchSnapshot();
    });

    it('should match the snapshot with optional properties', () => {
      scope.props.color = '#000';
      scope.props.hoverColor = '#fff';
      scope.props.size = '50px';

      expect(shallow(<BannerLogoSvg {...scope.props} />)).toMatchSnapshot();
    });
  });
});
