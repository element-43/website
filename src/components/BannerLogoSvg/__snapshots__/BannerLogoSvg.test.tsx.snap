// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BannerLogoSvg /> <BannerLogoSvg /> snapshots should match the snapshot with optional properties 1`] = `ShallowWrapper {}`;

exports[`<BannerLogoSvg /> <BannerLogoSvg /> snapshots should match the snapshot with the default properties 1`] = `ShallowWrapper {}`;
